pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -207,9 +208,20 @@ func (r *MigPlan) ListMigrations(client k8sclient.Client) ([]*MigMigration, erro
 
 // Registry label for controller-created migration registry resoruces
 const (
-	MigrationRegistryLabel = "migration-registry"
+	MigrationRegistryLabel        = "migration-registry"
+	MigrationRegistryDefaultImage = "registry:2"
+	MigrationRegistryImageEnvVar  = "MIGRATION_REGISTRY_IMAGE"
 )
 
+// Return the image reference for the migration registry (defaults to "registry:2")
+func migRegistryImageRef() string {
+	envImage := os.Getenv(MigrationRegistryImageEnvVar)
```

Comment:
It would be better to be consistent with other environment variables and put this logic in the `settings` package.  That said, we can merge as-is and make this change in 1.2. or 1.1.z.